### DISCUSSION TITLE
how to change the hightlighting style in my theme file

### DISCUSSION THREAD

**Discussion by sytpb** [PrismJS/prism#3520] [Q&A] [answered]
my next.js project with react . 

babelrc setup 

    "plugins": [
      [
        "prismjs",
        {
          "languages": ["javascript", "css", "markup"],
          "plugins": ["line-numbers"],
          "theme": "twilight",
          "css": true
        }
      ]
    ]

I want some token be changed with my customization style , such as font-size 18px ,  color: my color . 

i changed the file       node_modules\prismjs\themes\prism-twilight.css 

![image](https://user-images.githubusercontent.com/12178686/185025152-01565f0e-be2b-4ab1-a485-8bad7f9cde1b.png)

but it doesn't work for me .

**Comment by RunDevelopment** [comment, marked as answer]
To modify a theme, just copy the CSS file into your project directory and call it something like `custom-theme.css`, make your modifications to `custom-theme.css`, and then import it.

**Reply by sytpb** [reply]
![image](https://user-images.githubusercontent.com/12178686/185134592-28c5fdce-d678-4f83-90cd-69eab38fbe24.png)

![image](https://user-images.githubusercontent.com/12178686/185135095-54c9971c-8e94-42a9-8f3f-da8eeff4b3ff.png)

nothing modified except font-size .  

![image](https://user-images.githubusercontent.com/12178686/185136962-a7e5fb8b-1d3d-4ca5-8cae-966c14cd8056.png)

**Reply by RunDevelopment** [reply]
Prism themes are not CSS modules, so don't use the extension `.module.css`.

**Reply by sytpb** [reply]
ok , i found the root cause .    css file name change to global mode, such as my-prism.css ,    and import in _app.js ， and it worked .

**Reply by sytpb** [reply]
> Prism themes are not CSS modules, so don't use the extension `.module.css`.

thanks a lot .

**Reply by sytpb** [reply]
a little problem .   can't change the token's color ?.   changing font-size runs ok .

**Reply by sytpb** [reply]
![image](https://user-images.githubusercontent.com/12178686/185147717-94b78838-60b4-4794-874c-69e281e0c3cf.png)



```
 .token.tag,
 .token.boolean,
 .token.number,
 .token.deleted {
     color: white; /* #CF6A4C #f472b6*/
     font-size: 24px;
 }

```

**Reply by RunDevelopment** [reply]
This might be because of the order in which you import the types. Be sure to import your CSS file after Prism's. Otherwise, Prism's theme will override your styles. [Here's a good article explaining it in full](https://vecta.io/blog/definitive-guide-to-css-styling-order).

**Reply by sytpb** [reply]
and i render two code snippet , html works , javascript not work . 

```
const wrapper = () => {
  return (
    <div>
      <Code code={htmlCode} language="html" />
      <Code code={JSCode} language="javascript" />
    </div>
  )
};
```
![image](https://user-images.githubusercontent.com/12178686/185153347-41b631a8-a852-4077-824d-2afbabbead17.png)

**Reply by RunDevelopment** [reply]
I'm sorry, but I don't think that I can help you with that. This isn't an issue with Prism itself, but with your styles/project setup. I've never worked with Next.JS before, so I can't even make guesses as to what the problem might be.

**Reply by sytpb** [reply]
Yeah , this isn't an issue for prism .    Nice for you , you really help me a lot . thanks a again .
